templates/drawer_layout/app/controllers/Drawer.js

```diff
@@ -0,0 +1,3 @@
+function toggleDrawer(e) {
+	$.drawer.toggleLeft();
```

Comment:
Can you lint with the eslint-alloy preset? @ewanharris Did we add the .eslintrc to Alloy templates already as well? Would be good to have it in all templates, incl. instructions on how to configure eslint as part of the readme.